File: csharp/data-structures/Queue.cs
using System;
using System.Collections;

class MyQueue
{
    private Queue queue;
    public MyQueue()
    {
        queue = new Queue();
    }
    public void Add(string val)
    {
        queue.Enqueue(val);
    }
    public void Remove()
    {
        queue.Dequeue();
    }
    public void Clear()
    {
        queue.Clear();
    }
    public void PrintAll()
    {
        foreach (var elem in queue)
        {
            Console.WriteLine(elem);
        }
    }
}

File: python/data-structures/queue.py
class MyQueue:
    def __init__(self, data):
        self.data = data

    def enqueue(self, val):
        self.data.append(val)

    def dequeue(self):
        self.data.pop(0)

data = MyQueue([])
data.enqueue("a")
data.enqueue("b")
data.enqueue("c")

data.dequeue()

print(data.data)

# Iterate data
for i in data.data:
  print(i)
